pam.py

```diff
@@ -75,8 +75,9 @@ def authenticate(self, username, password, service='login', encoding='utf-8', re
 
         def conv(pam_self, query_list, user_data):
             response = []
-            for prompt, msg in query_list:
-                if msg == PAM.PAM_PROMPT_ECHO_OFF:
+            for index, query in enumerate(query_list):
+                prompt, msg = query
```

Comment:
do you mind changing this to the following form and removing the line that follows?

```python
for index, (prompt, msg) in enumerate(query_list):
```

Reply:
Fixed, thanks.